Auth0/Authentication.swift

```diff
@@ -1278,8 +1264,6 @@ public extension Authentication {
         .start { print($0) }
      ```
 
-     - precondition: if you are not using OAuth 2.0 API Authorization please use `delegation(parameters:)`
```

Comment:
`delegation(parameters:)` is deprecated, so we'll be removing it in a future PR.